packages/cli/src/lib/pdf/paged.js

```diff
@@ -9,15 +9,20 @@ export default async (input, options) => {
    * Configure the Paged.js Printer options
    * @see https://gitlab.coko.foundation/pagedjs/pagedjs-cli/-/blob/main/src/cli.js
    */
-  const printerOptions = { debug: options.debug || false }
+  const printerOptions = {
+    allowLocal: true,
+    debug: options.debug || false,
+  }
 
   const printer = new Printer(printerOptions)
 
   /**
    * Configure the Paged.js PDF options
    * @see
```

Comment:
@fchasen what is the most appropriate link to Paged.js PDF options documentation to use for the JSDoc here?